src/Bigcommerce/Api/Resources/Sku.php

```diff
@@ -19,17 +19,6 @@ class Sku extends Resource
         'product_id',
     );
 
-    public function options()
```

Comment:
I'd look at keeping this method for backwards compaitability (assuming it returns the same objects), with the implementation @ransomcarroll suggested here: https://github.com/bigcommerce/bigcommerce-api-php/pull/146#issuecomment-135205523